src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -16,11 +16,11 @@ import scala.concurrent.ExecutionContext
 class LeoRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest {
 
   class TestLeoRoutes(leonardoService: LeonardoService)
-                     (override implicit val materializer: Materializer, override val executionContext: ExecutionContext)
-    extends LeoRoutes(leonardoService, SwaggerConfig())(materializer, executionContext)
+                     (override implicit val materializer: Materializer, override implicit val executionContext: ExecutionContext)
+    extends LeoRoutes(leonardoService, SwaggerConfig())
```

Comment:
I think I changed this in my PR, but you omit the `override implicit`s:
```
class TestLeoRoutes(leonardoService: LeonardoService) extends LeoRoutes(leonardoService, SwaggerConfig())
```
and they'll still get wired correctly.